drivers/include/at.h

```diff
@@ -451,7 +489,7 @@ int at_parse_resp(at_dev_t *dev, char const *resp);
  * @param[in]   dev         device to operate on
  * @param[in]   resp_buf    buffer to store line
  * @param[in]   len         size of @p resp_buf
- * @param[in]   keep_eol    true to keep the CR character in the response
+ * @param[in]   keep_eol    true to keep any EOL sequence in the response
```

Comment:
The decision to keep only the first character of the EOL sequence seemed to be quite arbitrary and made distinction between some edge cases  a pain. So now you can either have the whole EOL sequence or not at all.